src/Data/Connection/PostObjectConnectionResolver.php

```diff
@@ -41,7 +41,14 @@ public function __construct( $source, $args, $context, $info, $post_type ) {
 		/**
 		 * Set the post type for the resolver
 		 */
-		$this->post_type = $post_type;
+		if ( 'revision' === $post_type || 'attachment' === $post_type ) {
+			$this->post_type = $post_type;
+		} else {
+			$post_type = is_array( $post_type ) ? $post_type : [ $post_type ];
+			unset(  $post_type['attachment'] );
+			unset( $post_type['revision'] );
+			$this->post_type = $post_type;
+		}
```

Comment:
An inline comment for this whole block would be good, it's a little confusing for why this is here.

Reply:
good call